toolkits/global/packages/global-forms/scss/00-tokens/_default.tokens.scss

```diff
@@ -31,7 +31,16 @@ $global-forms-label-color: #000000;
 $global-forms-label-font-size: 1rem;
 $global-forms-label-font-weight: 700;
 $global-forms-line-height: 1.4;
+$global-forms-label-inline-font-weight: 400;
 $global-forms-message-color-error: #c40606;
 $global-forms-message-icon-color-error: #c40606;
 $global-forms-message-icon-color-resting: #000000;
-$global-forms-message-icon-size: 1rem;
\ No newline at end of file
+$global-forms-message-icon-size: 1rem;
+$global-forms-pictographic-radio-icon-color-background-checked: #000000;
```

Comment:
This should be the `#01324b` value, not black, to match the other checked color?

Reply:
As yes, thank you. I'd forgotten to investigate the checkbox Sass variables for this.

Reply:
FWIW you can usually go for very low precision without any visible degradation, in my experience.

Reply:
I have updated the component's design tokens so that these settings related to the checked and unchecked state of pictographic radios follows the same structure and naming convention as has been used for checkboxes. As you suggested I have updated these state colours to use `{color.primary.universal-dark-blue}` (`#01324b`) instead of black.